gc/base/SparseAddressOrderedFixedSizeDataPool.hpp

```diff
@@ -170,6 +183,15 @@ class MM_SparseAddressOrderedFixedSizeDataPool : public MM_BaseVirtual
 	 * @return true if data pointer is valid
 	 */
 	bool isValidDataPtr(void *dataPtr);
+	/**
+	 * Check if the given data pointer is valid and consistent(dataPtr, proxyObjPtr and size)
+	 *
+	 * @param dataPtr		void*						Data pointer
+	 * @param proxyObjPtr	void*						Proxy object associated with dataPtr
+	 * @param size			uintptr_t					Size of region consumed by dataPtr
```

Comment:
Formatting is malformed. Using spaces should resolve the issue. The same issue is also seen for the method descriptions below.